test/lib/builder/builder.js

```diff
@@ -53,14 +53,16 @@ function cloneProjectTree(tree) {
 async function checkFileContentsIgnoreLineFeeds(expectedFiles, expectedPath, destPath) {
 	for (let i = 0; i < expectedFiles.length; i++) {
 		const expectedFile = expectedFiles[i];
-		const relativeFile = path.relative(expectedPath, expectedFile);
-		const destFile = path.join(destPath, relativeFile);
-		const currentFileContentPromise = readFile(destFile, "utf8");
-		const expectedFileContentPromise = readFile(expectedFile, "utf8");
-		const assertContents = ([currentContent, expectedContent]) => {
-			assert.equal(currentContent.replace(newLineRegexp, "\n"), expectedContent.replace(newLineRegexp, "\n"));
-		};
-		await Promise.all([currentFileContentPromise, expectedFileContentPromise]).then(assertContents);
+		if (!expectedFile.endsWith(".zip")) {
```

Comment:
There is a separate check for the content of the ZIP file - but we omit to 1..1 compare the ZIP file content in that exec here.

Reply:
This is my point. Since there is a separate check, we should not test the `generateManifestBundle` task here as well. Only testing for the *existence* of  the ZIP files does not sufficiently test the task, so rather not test it at all. Otherwise we cover code without testing it since most of the tasks code is about the *content* of the ZIP file.

Reply:
Still to be discussed

Reply:
Is this change still needed? The builder tests don't produce zip files anymore, now that the task is not executed by default anymore.